NickvisionTagger.Shared/org.nickvision.tagger.desktop.in

```diff
@@ -7,6 +7,8 @@ Icon=org.nickvision.tagger
 Terminal=false
 Type=Application
 Categories=AudioVideo;
```

Comment:
```suggestion
Categories=AudioVideo;Audio;
```

Reply:
Should we just use `Audio` instead of `AudioVideo` since Tagger only supports music files?

Reply:
No, `AudioVideo` is base category, `Audio` is specific

Reply:
https://specifications.freedesktop.org/menu-spec/latest/apa.html

>  Audio | An audio application | Desktop entry must include AudioVideo as well